### PR TITLE
pwntools 2.2.0 (new formula)

### PR DESCRIPTION
**Pull request by DomT4**
There’s probably an ethical discussion to be had here, but here’s a formula for the “pwntools” toolkit. The trillion resources makes it suited to being handled by Homebrew, as does the small mountain of executables generated, but I won’t ram it down maintainer’s throats unless they’re happy to ship such a toolkit.

### CODE REVIEW

**Comment by DomT4** [comment]
That's a weird error. It seems to be having some kind of curses issue. Can't reproduce at all locally. I'm using Homebrew's Python rather than the system one though - Could that be the failure here @tdsmith?

**Comment by tdsmith** [comment]
I'd try adding `ENV["TERM"] = "xterm"` at the top of the `test` block.

**Comment by DomT4** [comment]
Thanks for the suggestion! Have pushed, shall see how the bot feels now :smile_cat:

File: Library/Formula/pwntools.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    ENV["TERM"] = "xterm"
+    assert_equal "686f6d6562726577696e7374616c6c636f6d706c657465",
+    shell_output("#{bin}/hex homebrewinstallcomplete").strip
```
Comment (mikemcquaid) [inline Library/Formula/pwntools.rb]: May as well use `system` here if we're not checking output or using a non-zero exit code.

Comment (mikemcquaid) [inline Library/Formula/pwntools.rb]: Ah, whoops, misread due to indentation. Can you indent this line two spaces?

Comment (DomT4) [inline Library/Formula/pwntools.rb]: Ah, sure. Apologies, I got overly anal and lined them up alongside each other nicely, and didn't immediately realise it made reading the test harder for people. Will fix :smile:

Comment (DomT4) [inline Library/Formula/pwntools.rb]: Fixed :)

**Pull request closed without merging**